doc/api/packages.md

```diff
@@ -704,7 +704,6 @@ is provided below to assist with ecosystem coordination.
 
 * `"types"` - can be used by typing systems to resolve the typing file for
   the given export. _This condition should always be included first._
-* `"deno"` - indicates a variation for the Deno platform.
 * `"browser"` - any web browser environment.
 * `"react-native"` - will be matched by the React Native framework (all
   platforms). _To target React Native for Web, `"browser"` should be specified
```

Comment:
> Remove react-native?

I suggest we do, it's already listed at https://runtime-keys.proposal.wintercg.org/#react-native

```suggestion
```